### PR TITLE
Single video page

### PR DESCRIPTION
**Pull request by varnita619**
added 
- single video page
- add to history functionality
- remove from history functionality
- clear all history functionality

Screenshot (38)

### CODE REVIEW

File: src/Components/SideBar/SideBar.jsx [hunk 1]
```diff
@@ -9,11 +9,11 @@ import { useAuthContext } from "../../Context/AuthContext";
 
 function SideBar() {
   const navLink = ({ activeLink }) => ({
-    borderBottom: activeLink ? '.2rem solid var(--secondary-color)' : '',
+    borderBottom: activeLink ? ".2rem solid var(--secondary-color)" : "",
```
Comment (surajmore98) [inline src/Components/SideBar/SideBar.jsx:12]: you can create class for ".2rem solid var(--secondary-color)" use that in condition.

File: src/Components/SideBar/SideBar.jsx [hunk 2]
```diff
@@ -9,11 +9,11 @@ import { useAuthContext } from "../../Context/AuthContext";
...
   });
   const getActiveStyle = ({ isActive }) => ({
-    backgroundColor: isActive ? "red" : ""
-  })
+    backgroundColor: isActive ? "red" : "",
```
Comment (surajmore98) [inline src/Components/SideBar/SideBar.jsx:15]: you can do same for this too.

**Review by surajmore98 — APPROVED** [review]
clean,  easy understandable and well structured code
good to merge.

**Review by anuj-kumary — APPROVED** [review]

**Review by mdarif6 — APPROVED** [review]
- At video listing page side bar- you can remove underline from link items.
- folder structure is good.
- other suggestion you got from above pr.

**Pull request merged**